scripts/generate_nwb.py

```diff
@@ -6,6 +7,9 @@
 PWD = os.path.dirname(os.path.abspath(__file__))
 USER_HOME = os.path.expanduser("~")
 
+logging.config.fileConfig(fname=str(PWD) + '/../nsds_lab_to_nwb/logging.conf', disable_existing_loggers=False)
```

Comment:
I think we'll eventually want to do this in a different way, I'll make an issue.

Reply:
Sounds good, look forward to seeing your solution - if there's a better way to apply fileConfig that would be nice to know.